Lib9c/Model/State/ArenaInfomation.cs

```diff
@@ -0,0 +1,311 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Bencodex.Types;
+using Libplanet;
+using Nekoyume.Battle;
+using Nekoyume.Model.BattleStatus;
+using Nekoyume.Model.Item;
+using Nekoyume.TableData;
+
+namespace Nekoyume.Model.State
+{
+    public class ArenaInfomation : IState
```

Comment:
The `Model.State.ArenaInfomation` and the `Model.Arena.ArenaInfomation` look too similar.
This `Model.State.ArenaInfomation` was `Model.State.ArenaInfo` right? If so it is better to use origin name than new name I think.